api/package.json

```diff
@@ -22,11 +22,13 @@
     "bowser": "^2.11.0",
     "buffer-shims": "^1.0.0",
     "compression": "^1.7.4",
+    "cron-validator": "^1.3.1",
     "express": "^4.18.2",
     "google-libphonenumber": "^3.2.33",
     "gsm": "^0.1.4",
     "helmet": "^3.23.3",
     "http-proxy": "^1.18.1",
+    "later": "^1.2.0",
```

Comment:
I _think_ we only need these in the shared library. So both `later` and `cron-validator` can be removed from `api` as dependencies.

Reply:
I tried to remove `later `and `cron-validator` from `api/package.json` and `sentinel/package.json` but got this error in each case:

```
2023-09-28 09:47:59 2023-09-28T13:47:59.960 ERROR: Fatal error intialising sentinel 
2023-09-28 09:47:59 2023-09-28T13:47:59.961 ERROR: Error: Error loading configuration
2023-09-28 09:47:59     at /service/sentinel/src/config.js:75:13
2023-09-28 09:47:59     at processTicksAndRejections (node:internal/process/task_queues:96:5) {
2023-09-28 09:47:59   [stack]: 'Error: Error loading configuration\n' +
2023-09-28 09:47:59     '    at /service/sentinel/src/config.js:75:13\n' +
2023-09-28 09:47:59     '    at processTicksAndRejections (node:internal/process/task_queues:96:5)',
2023-09-28 09:47:59   [message]: 'Error loading configuration'
2023-09-28 09:47:59 } 
2023-09-28 09:48:13 Info: Starting CHT Sentinel
2023-09-28 09:48:14 2023-09-28T13:48:14.437 INFO: Running server checks... 
2023-09-28 09:48:14 Node Environment Options: 'undefined'
2023-09-28 09:48:14 Node Version: 16.20.1 in development mode
2023-09-28 09:48:14 CouchDB Version: 3.3.2
2023-09-28 09:48:14 CouchDb Cluster ready
2023-09-28 09:48:14 2023-09-28T13:48:14.647 INFO: API is ready: {"ready":true,"handler":"medic-api","version":"0.1.0","detail":"All required services are running normally"} 
2023-09-28 09:48:15 2023-09-28T13:48:15.223 ERROR: Error: Cannot find module 'cron-validator'
2023-09-28 09:48:15 Require stack:
2023-09-28 09:48:15 - /service/shared-libs/transitions/src/lib/utils.js
2023-09-28 09:48:15 - /service/shared-libs/transitions/src/transitions/index.js
2023-09-28 09:48:15 - /service/shared-libs/transitions/src/index.js
2023-09-28 09:48:15 - /service/sentinel/src/config.js
2023-09-28 09:48:15 - /service/sentinel/server.js
2023-09-28 09:48:15     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1028:15)
2023-09-28 09:48:15     at Function.Module._load (node:internal/modules/cjs/loader:873:27)
2023-09-28 09:48:15     at Module.require (node:internal/modules/cjs/loader:1100:19)
2023-09-28 09:48:15     at require (node:internal/modules/cjs/helpers:108:18)
2023-09-28 09:48:15     at Object.<anonymous> (/service/shared-libs/transitions/src/lib/utils.js:3:14)
2023-09-28 09:48:15     at Module._compile (node:internal/modules/cjs/loader:1198:14)
2023-09-28 09:48:15     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
2023-09-28 09:48:15     at Module.load (node:internal/modules/cjs/loader:1076:32)
2023-09-28 09:48:15     at Function.Module._load (node:internal/modules/cjs/loader:911:12)
2023-09-28 09:48:15     at Module.require (node:internal/modules/cjs/loader:1100:19) {
2023-09-28 09:48:15   [stack]: "Error: Cannot find module 'cron-validator'\n" +
2023-09-28 09:48:15     'Require stack:\n' +
2023-09-28 09:48:15     '- /service/shared-libs/transitions/src/lib/utils.js\n' +
2023-09-28 09:48:15     '- /service/shared-libs/transitions/src/transitions/index.js\n' +
2023-09-28 09:48:15     '- /service/shared-libs/transitions/src/index.js\n' +
2023-09-28 09:48:15     '- /service/sentinel/src/config.js\n' +
2023-09-28 09:48:15     '- /service/sentinel/server.js\n' +
2023-09-28 09:48:15     '    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1028:15)\n' +
2023-09-28 09:48:15     '    at Function.Module._load (node:internal/modules/cjs/loader:873:27)\n' +
2023-09-28 09:48:15     '    at Module.require (node:internal/modules/cjs/loader:1100:19)\n' +
2023-09-28 09:48:15     '    at require (node:internal/modules/cjs/helpers:108:18)\n' +
2023-09-28 09:48:15     '    at Object.<anonymous> (/service/shared-libs/transitions/src/lib/utils.js:3:14)\n' +
2023-09-28 09:48:15     '    at Module._compile (node:internal/modules/cjs/loader:1198:14)\n' +
2023-09-28 09:48:15     '    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)\n' +
2023-09-28 09:48:15     '    at Module.load (node:internal/modules/cjs/loader:1076:32)\n' +
2023-09-28 09:48:15     '    at Function.Module._load (node:internal/modules/cjs/loader:911:12)\n' +
2023-09-28 09:48:15     '    at Module.require (node:internal/modules/cjs/loader:1100:19)',
2023-09-28 09:48:15   [message]: "Cannot find module 'cron-validator'\n" +
2023-09-28 09:48:15     'Require stack:\n' +
2023-09-28 09:48:15     '- /service/shared-libs/transitions/src/lib/utils.js\n' +
2023-09-28 09:48:15     '- /service/shared-libs/transitions/src/transitions/index.js\n' +
2023-09-28 09:48:15     '- /service/shared-libs/transitions/src/index.js\n' +
2023-09-28 09:48:15     '- /service/sentinel/src/config.js\n' +
2023-09-28 09:48:15     '- /service/sentinel/server.js',
2023-09-28 09:48:15   code: 'MODULE_NOT_FOUND',
2023-09-28 09:48:15   requireStack: [
2023-09-28 09:48:15     '/service/shared-libs/transitions/src/lib/utils.js',
2023-09-28 09:48:15     '/service/shared-libs/transitions/src/transitions/index.js',
2023-09-28 09:48:15     '/service/shared-libs/transitions/src/index.js',
2023-09-28 09:48:15     '/service/sentinel/src/config.js',
2023-09-28 09:48:15     '/service/sentinel/server.js',
2023-09-28 09:48:15     [length]: 5
2023-09-28 09:48:15   ]
2023-09-28 09:48:15 } 
2023-09-28 09:48:15 2023-09-28T13:48:15.224 ERROR: Fatal error intialising sentinel 
2023-09-28 09:48:15 2023-09-28T13:48:15.228 ERROR: Error: Error loading configuration
2023-09-28 09:48:15     at /service/sentinel/src/config.js:75:13
2023-09-28 09:48:15     at processTicksAndRejections (node:internal/process/task_queues:96:5) {
2023-09-28 09:48:15   [stack]: 'Error: Error loading configuration\n' +
2023-09-28 09:48:15     '    at /service/sentinel/src/config.js:75:13\n' +
2023-09-28 09:48:15     '    at processTicksAndRejections (node:internal/process/task_queues:96:5)',
2023-09-28 09:48:15   [message]: 'Error loading configuration'
2023-09-28 09:48:15 } 
```

Reply:
:( workspaces are just weird